core/controllers/library.py

```diff
@@ -73,6 +73,12 @@ class LibraryPage(base.BaseHandler):
     def get(self):
         """Handles GET requests."""
         search_mode = 'search' in self.request.url
+        rank_mode = 'explorations' in self.request.url
+
+        if search_mode:
```

Comment:
Right now I think the {{ title }} will automatically be translated once I make a i18n variables for the strings. For example the title for the main library page will automatically be translated because I am passing a i18n variable for it. 
Look at this comment in `base.html` on line 62:
```
<!-- The title is bound to the rootScope. The content of the block
maintitle can be a string or a translation id. If it is a translation it
will be replaced by its translation when the page is loading. If it is a
string it would be displayed as is. This is the only way to translate
the page title because the head of the file is outside the scope of
any other controller. -->
```
So I think to solve this problem we have to translate the strings and make i18n variables for them (which can be done in a seperate PR). Is it okay if I pass the title directly from the backend?
What are your views on this?